src/service/url-replacements-impl.js

```diff
@@ -394,12 +446,41 @@ export class UrlReplacements {
    * @return {!Promise<string|undefined>}
    * @private
    */
-  getTimingData_(startEvent, endEvent) {
+  getTimingDataAsync_(startEvent, endEvent) {
+    const syncMetric = this.getTimingDataSync_(startEvent, endEvent);
+    if (syncMetric === '') {
+      // Metric is not yet available. Retry after a delay.
+      return loadPromise(this.win_).then(() => {
+        const timingInfo = this.win_['performance']
```

Comment:
Is this essentially the same as `#getTimingDataSync_`? If so, can't we just call that again now that the window has finished loading?